README.md

```diff
@@ -6,6 +6,44 @@ This repository contains [actions](https://docs.github.com/en/free-pro-team@late
 
 Runs license and python linting checks.
 
+✔️ [symbiflow/actions/update_conda_lock](update_conda_lock)
+
+Simplifies maintaining Conda Locks, i.e. stable Conda environment files with precise Conda and Pip package versions based on a given Conda environment file.
+
+Running this action creates a Pull Request that creates or updates a Conda Lock.
+Checking workflow results and merging the Pull Request are the only actions that require repository maintainer's attention.
+
+Please bear in mind that the Pull Request can only trigger workflows if this action is used with GitHub's [Personal Access Token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token).
```

Comment:
FYI - A deploy keys with push access is a better solution than PATs as they only give you access to a single repository.

Reply:
It's a nice option!